itests/eth_hash_lookup_test.go

```diff
@@ -83,8 +83,15 @@ func TestTransactionHashLookup(t *testing.T) {
 	hash := client.EVM().SubmitTransaction(ctx, &tx)
 	require.Equal(t, rawTxHash, hash)
 
-	mpoolTx, err := client.EthGetTransactionByHash(ctx, &hash)
-	require.NoError(t, err)
+	var mpoolTx *ethtypes.EthTx
+	for i := 0; i < 3; i++ {
```

Comment:
ditto question here, ISTM that this should pass the first time as we have `EthSendRawTransaction` wired up to synchronously insert into the db